skel/.vimrc

```diff
@@ -0,0 +1,18 @@
+set number
+set smartcase
+set autoindent
+set nowrap
+set incsearch
+set hlsearch
+set cursorline
```

Comment:
Questionable effect depending on the colour scheme and/or terminal. I'd take this out.

Reply:
OK.